src/components/Buttons/DoneButton/DoneButton.tsx

```diff
@@ -9,7 +9,7 @@ interface DoneButtonProps {
 
 const DoneButton = ({ active, onClick }: DoneButtonProps) => (
   <button
-    onClick={onClick}
+    onClick={(e) => (active ? onClick(e) : () => false)}
```

Comment:
onClick={(e) => (active && onClick(e)}

why do you have to return 'false' here